src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechRecognizer.java

```diff
@@ -572,4 +560,29 @@ public void unsetWakeWord() {
     setWakeWord(null);
   }
 
+  @Override
+  public ServiceConfig getConfig() {
+    WebkitSpeechRecognitionConfig config = (WebkitSpeechRecognitionConfig) initConfig(new WebkitSpeechRecognitionConfig());

```

Comment:
Should be AbstractSpeechRecognizerConfig not webkit..

Reply:
oop yep - fixed